ArcGIS-toolbox/ASF_Tools.pyt

```diff
@@ -838,13 +853,18 @@ class RGBDecomp(object):
             xps = arcpy.sa.Square(xp)
             arcpy.AddMessage("Input scale is amplitude. Converted to power scale.")
 
+        elif scale == 'dB':
+            cps = arcpy.sa.Power(10, arcpy.sa.Divide(cp, 10))
+            xps = arcpy.sa.Power(10, arcpy.sa.Divide(xp, 10))
```

Comment:
We explicitly do not chain spatial analysis function like this in the scale conversion tool, so we may need to do these in steps, but I'm not sure if that's a technical limitation or style.

Reply:
I was probably using an abundance of caution in the scale conversion tool.
The RGB outputs from input datasets in power are identical to those from input datasets in dB and amplitude, so this can probably be left as-is.

Reply:
Being separate above def. makes it easier to read, so I think that was a good style choice. Glad to know this works!